content/projects/kubernetes/kubernetes-manifest-generator.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Levis: Kubernetes Manifest Generator'
+showMetadata: true
+editable: true
+showToc: true
+categories:
+  - project
+tags:
+  - kubernetes
+---
+
+Original Author: [Sirinat Paphatsirinatthi](https://github.com/dmakeroam)
+Source Code: https://github.com/kubeopsskills/levis
+
+The aim of this project is to simplify Kubernetes configuration.
```

Comment:
The aim of this project is to simplify Kubernetes configuration.
You just write a few lines, then deploy the configuration to Kubernetes cluster